packages/sync/src/class-item.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Item is a generic class which helps us bring uniformity to data that we sent.
+ *
+ * @package automattic/jetpack-sync
+ */
+
+namespace Automattic\Jetpack\Sync;
+
+/**
+ * Sync Item class lets us bring uniformity to the Sync Data that we send.
+ */
+class Item {
```

Comment:
Instead of all these getters and setters for state, why not just have props? Or if you _really_ need to intercept props with a function, use `__get` and `__set` magic methods.